test/plain-list/AutosuggestApp.test.js

```diff
@@ -406,6 +730,29 @@ describe('Plain list Autosuggest', () => {
     });
   });
 
+  describe('inputProps.onChange with selectFirstSuggestion', () => {
+    before(() => {
+      setSelectFirstSuggestion(true);
+    });
+
+    after(() => {
+      setSelectFirstSuggestion(false);
+    });
+
+    beforeEach(() => {
+      onChange.reset();
+    });
+
+    it('should be called with the right parameters when user types', () => {
+      focusAndSetInputValue('c');
+      expect(onChange).to.have.been.calledOnce;
+      expect(onChange).to.be.calledWith(eventInstance, {
+        newValue: 'c',
+        method: 'type'
+      });
+    });
+  });
```

Comment:
How is this test different from [this one](https://github.com/moroshko/react-autosuggest/blob/master/test/plain-list/AutosuggestApp.test.js#L317-L324)?